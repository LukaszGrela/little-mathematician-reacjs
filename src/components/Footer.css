footer {
    position: absolute;
    bottom: 0;
    width: 100%;
    padding: 10px 0;
    
}