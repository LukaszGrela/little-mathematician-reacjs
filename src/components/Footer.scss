
footer {
    position: absolute;
    bottom: 0;
    width: 100%;
    padding: 10px 0;
    font-size: 0.8em;
}
@media only screen 
and (max-device-height : 668px)
and (orientation: landscape)
{
    footer {
        position: initial;
    }
    
}