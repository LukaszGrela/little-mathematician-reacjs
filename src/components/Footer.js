import React, { Component } from 'react';

import './Footer.css'

class Footer extends Component {
    state = {}
    render() {
        return (
            <footer>GrelaDesign (c) 2017</footer>
        );
    }
}

export default Footer;