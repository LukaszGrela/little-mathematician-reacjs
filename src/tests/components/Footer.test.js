import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer';

test('should render Footer correctly', () => {

    const wrapper = shallow(<Footer />);
    wrapper.setState({ currentYear: 2018 });
    expect(wrapper).toMatchSnapshot();
});

test('should render Footer correctly with different year', () => {

    const wrapper = shallow(<Footer />);
    wrapper.setState({ currentYear: 2019 });
    expect(wrapper).toMatchSnapshot();
})