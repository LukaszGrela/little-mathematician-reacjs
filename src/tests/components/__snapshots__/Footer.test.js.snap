// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer correctly 1`] = `
<footer>
  GrelaDesign (c) 2017 - 
  2018
</footer>
`;

exports[`should render Footer correctly with different year 1`] = `
<footer>
  GrelaDesign (c) 2017 - 
  2019
</footer>
`;
